packages/katello/katello/backup.rb

```diff
@@ -372,6 +371,11 @@ def validate_directory
       end
     end
 
+    def validate_services
+      result = run_cmd("katello-service status #{@excluded}", [0,1])
+      result.include?('dead') ? false : true
```

Comment:
`result.include?('dead')` should return true/false on its own

Reply:
Yes, but this is backwards. So if services are dead, that means the services are down and thus validate_services would be false.

Reply:
I could negate it in one line, but I believe this is the chosen format rather than `not` or `!`

Reply:
I've never seen this pattern used before, do you know of any docs or examples that say/use ternary true/false is used over `!my_condition`?

Reply:
I'm happy to change it, I prefer it that way. I recall it from the katello dev guide a while back, but no longer see it. Might have been a different style guide altogether.